.github/workflows/release.yml

```diff
@@ -57,6 +57,9 @@ jobs:
         run: |
           export FORCE_CUDA=1
           export TORCH_CUDA_ARCH_LIST="3.5;5.0+PTX;6.0;7.0;7.5;8.0;8.6"
+          TORCH_VERSION=`echo "pt${{ matrix.torch-version }}" | sed "s/..$//" | sed "s/\.//g"`
```

Comment:
`..$` is a little brittle because it will always remove the last two digits. If you want to remove the patch version and any suffixes, i.e. just get the major and minor version, then `'s/^\([0-9]\+\.[0-9]\+\).*$/\1/'` is an option.

Reply:
Thanks! Have fixed